CHANGELOG.md

```diff
@@ -81,6 +81,9 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 - [Discover] Fix missing index pattern field from breaking Discover [#5626](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/5626)
 - [BUG] Remove duplicate sample data as id 90943e30-9a47-11e8-b64d-95841ca0b247 ([5668](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/5668))
 - [BUG][Multiple Datasource] Fix datasource testing connection unexpectedly passed with wrong endpoint [#5663](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/5663)
+- [Table Visualization] Fix filter action buttons for split table aggregations ([#5619](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/5619))
```

Comment:
It seems like that you include the others changes here.

Reply:
I didn't include it, but it is hanging here since it was left alone from the auto backport https://github.com/opensearch-project/OpenSearch-Dashboards/pull/5823/files, and original PR has the changelog https://github.com/opensearch-project/OpenSearch-Dashboards/pull/5619/files

If I remove it, it will not be in 2.x but it should. How should we handle it? @bandinib-amzn @ashwin-pc

Reply:
According to [Backport workflow](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/.github/workflows/backport.yml#L41), we skip changelog for backport PR. That's why we don't see changelog in autocut backport. I'm not sure if we use changelog from 2.x branch.

So for this PR, I would suggest keep only your changes in `CHANGELOG.md`. 

@ashwin-pc Can you tell us what is best practice for changelog when we manually backport PR?

Reply:
I removed the dangling changelog for now